projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -222,7 +222,7 @@
         [style.width.px]="scrollWidth"></div>
 </div>
 
-<div class="igx-grid__scroll" #scr [hidden]="isHorizontalScrollHidden">
+<div class="igx-grid__scroll" [style.height.px]="scrollWidth" #scr [hidden]="isHorizontalScrollHidden">
```

Comment:
Maybe we should rename `scrollWidth` to `scrollSize` since it is now used for both height and width.
The size calculation can be moved to the VirtualHelperBaseDirective.